components/form/form-element-mixin.js

```diff
@@ -153,6 +160,13 @@ export const FormElementMixin = superclass => class extends LocalizeCoreElement(
 		return this._validity;
 	}
 
+	firstUpdated(changedProperties) {
+		super.firstUpdated(changedProperties);
+		if (this.validateOnInit) {
+			this.requestValidate(true);
+		}
+	}
```

Comment:
Doing this is `firstUpdated` should be redundant if you're also doing it in `updated` (which gets called right after).